game-headed/src/main/java/org/triplea/game/client/ui/javafx/screen/RootActionPane.java

```diff
@@ -49,8 +49,11 @@ public void setContent(final Node node) {
     Preconditions.checkNotNull(node);
     Preconditions.checkState(Platform.isFxApplicationThread());
 
-    rootPane.getChildren().clear();
-    rootPane.getChildren().add(node);
+    if (root.getChildren().size() == 3) {
```

Comment:
What does `3` signify? Is it only relevant for the `Screens.CONTENT` case?

Reply:
Well there are supposed to be 3 different childs, but I can see that this isn't really clear.